src/main/java/uk/gov/service/notify/NotificationClient.java

```diff
@@ -191,6 +191,32 @@ public SendSmsResponse sendSms(String templateId, String phoneNumber, Map<String
 
 
     /**
+     * The sendSms method will create an HTTPS POST request. A JWT token will be created and added as an Authorization header to the request.
```

Comment:
Can you use 

/**
 * {@inheritDoc}
 */

As the interface docs are identical?